.scripts/fix_line_endings.py

```
@@ -0,0 +1,63 @@
+def to_lf(path):
+    try:
+        with open(path, "rb") as f:
+            content = f.read()
+
+        # Replace CRLF with LF
+        if b"\r\n" in content:
+            print(f"Fixing CRLF in {path}")
+            content = content.replace(b"\r\n", b"\n")
+            with open(path, "wb") as f:
+                f.write(content)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)